tox.ini

```diff
@@ -40,7 +40,7 @@ commands = isort --diff --check planar_ally/ tests/
 
 [testenv:safety]
 deps = safety
-whitelist_externals =
+allowlist_externals =
```

Comment:
CICD started failing with errors about these commands being disallowed w/o this change, see results from https://github.com/noteable-io/noteable-notebook-magics/actions/runs/3643223520